pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
```

Comment:
I'll mention it in util.FindFiles() but would much rather FindFiles() handle a globbing (or regex), rather than automatically assuming "contains" - That would let us search for only *.y.?ml, for example, cleaning up the code below.